src/app/app.component.ts

```diff
@@ -1,14 +1,35 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
 
 import { UserService } from '@shared/user.service';
+import { LoaderService } from '@shared/loader.service';
+import { MatDialog } from '@angular/material';
+import { LoaderComponent } from '@core/loader/loader.component';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
+  constructor(public userService: UserService, private loaderService: LoaderService, private dialog: MatDialog) {
+  }
+
+  ngOnInit(): void {
```

Comment:
To be consistent think to use `void` everywhere or never ;)

Reply:
Done.